sdk/confidentialledger/confidential-ledger-rest/src/confidentialLedger.ts

```diff
@@ -22,7 +22,7 @@ export interface ConfidentialLedgerClientOptions extends ClientOptions {
 export default function createClient(
   endpoint: string,
   credentials: TokenCredential,
-  { apiVersion = "2024-01-26-preview", ...options }: ConfidentialLedgerClientOptions = {},
+  { apiVersion = "2024-12-09-preview", ...options }: ConfidentialLedgerClientOptions = {},
```

Comment:
Hey we notice you report codegen issue [here](https://github.com/Azure/autorest.typescript/issues/3183#issuecomment-2857079162), does this issue come from this lib? could you provide more detail?

Reply:
Now that re-generated code is under `src/generated/`, it felts that we probably should remove this file and other generated files under `src/`, and re-export generated things from `src\generated\*.js` instead?  I suppose you didn't have any customization done on top of `src/isUnexpected.ts`, `src/clientDefinitions.ts`, etc.?